runtime/src/main/scala-3/mdoc/document/Binder.scala

```diff
@@ -0,0 +1,15 @@
+package mdoc.document
+
+final class Binder(val value: Any, val name: String, val pos: RangePosition) {
```

Comment:
`Binder`, `Document` and `DocumentBuilder` will most likely be merged back once pprint support is added.